tst/standard/attr.tst

```diff
@@ -316,6 +316,23 @@ gap> DigraphStronglyConnectedComponents(gr2);
 rec( comps := [ [ 1 ], [ 2 ], [ 3 ], [ 4 ], [ 5 ], [ 6 ], [ 7 ], [ 8 ], 
       [ 9 ], [ 10 ] ], id := [ 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 ] )
 
+#  DigraphNrStronglyConnectedComponents
+gap> D := CycleDigraph(10);;
+gap> for i in [1 .. 1000] do
```

Comment:
Can you please decrease this number to something much smaller, like `50` or something. For me this code takes almost half a second, and ideally we'd like each standard test file to take only about a second or two in total.